CODE_REVIEW.md

```diff
@@ -0,0 +1,117 @@
+# Pull Request (PR) Review Process
+
+## Introduction
+
+We perform a code review in order to ensure that the produced code meets the requirements of the QASP. That defines the deliverables, performance standards for each, the required quality level for each, and defines how each will be assessed at the end of each sprint. Although ultimately it is up to the contracting staff to monitor vendor performance, they rarely have the technical experience to do so for an Agile team, and rely on a developer—you—to evaluate compliance with performance standards on a per-sprint basis.
+
+Here are the Deliverables and Performance Standards for this project:
+
+| Deliverable | Performance Standard(s) | Acceptable Quality Level | Method of Assessment |
+| --- | --- | --- | --- |
+| Tested Code | Code delivered under the order must have substantial test code coverage<br><br>Version-controlled Joint Commissions GitHub repository of code that comprises product that will remain in the government domain | Minimum of 90% test coverage of all code. All areas of code are meaningfully  tested. | Combination of manual review and automated testing |
```

Comment:
I'm guessing the "<br><br>" appearing here is unintentional?

Reply:
It's intentional, but awkward! In the Markdown for a table, there's no pure-Markdown way to say "start a new paragraph"—hitting "enter" twice would tell Markdown "I'm all done with the table now." So the only solution is to drop in some HTML that accomplishes the same thing.